src/biokbase/narrative/tests/data/ee2_job_test_data.json

```diff
@@ -244,7 +244,7 @@
                     "workspace_name": "ialarmedalien:narrative_1623776017922"
                 }
             ],
-            "parent_job_id": "60e7112887b7e512a899c8f1",
+            "parent_job_id": null,
```

Comment:
removed these -- could probably get rid of them altogether at some point in the nearish future